docs/install.md

```diff
@@ -10,7 +10,7 @@
 
 ## Installation
 #### Set up driver permission
-The driver requires IAM permission to talk to Amazon EBS to manage the volume on user's behalf. [The example policy here](./example-iam-policy.json) defines these permissions. There are several methods to grant the driver IAM permission:
+The driver requires IAM permission to talk to Amazon EBS to manage the volume on user's behalf. [The example policy here](./example-iam-policy.json) defines these permissions. A [stricter example policy](./example-iam-policy-strict.json) is also provided, but needs to be modified before use. There are several methods to grant the driver IAM permission:
```

Comment:
Perhaps a note here as well that `<<CLUSTER_ID>>` needs to be updated to set the cluster ID value in the policy